### PR TITLE
vo draft Extraction workflows STO-6997

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
### Preview links
- [Extraction scans](https://6672dccf7ffbc42e05e236c0--harness-developer.netlify.app/docs/security-testing-orchestration/use-sto/orchestrate-and-ingest/extraction-scans)
- Anchore Enterprise
  - [Target](https://6672dccf7ffbc42e05e236c0--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/anchore-enterprise-scanner-reference#target) Added note at end: _In Extraction mode, the target name and variant should match the **image name** and [**image tag** below._
  - [Container image type](https://6672dccf7ffbc42e05e236c0--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/anchore-enterprise-scanner-reference#type-1) Added note that the scanned image must be on the server.

- SonarQube SonarScanner
   - [Scan configuration](https://6672dccf7ffbc42e05e236c0--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/sonarqube-sonar-scanner-reference#scan-configuration)
   - [Target name](https://6672dccf7ffbc42e05e236c0--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/sonarqube-sonar-scanner-reference#name) added note: _If you're running an Extraction scan, this field should match the code repository name in SonarQube._
   - [Target variant](https://6672dccf7ffbc42e05e236c0--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/sonarqube-sonar-scanner-reference#variant) added note: _If you're running an Extraction scan, this field should match the branch or PR defined in SonarQube._ 

- [STO 1.83 fixed issues](https://6672dccf7ffbc42e05e236c0--harness-developer.netlify.app/release-notes/security-testing-orchestration#fixed-issues-12) -- added note about feature flag
- Early Access features [SonarQube branch scanning](https://6672dccf7ffbc42e05e236c0--harness-developer.netlify.app/release-notes/early-access#sonarqube-branch-scanning) -- added section about branch scanning

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/security-testing-orchestration/sto-techref-category/shared/step_palette/target/auto-detect/_note.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Note the following:
-- **Auto** is not available when the **Scan Mode** is **Ingestion**. 
+- **Auto** is not available when the **Scan Mode** is **Ingestion** or **Ingestion**.
```
Comment (atmollo-harness) [inline docs/security-testing-orchestration/sto-techref-category/shared/step_palette/target/auto-detect/_note.md:2]: Ingestion or Ingestion?

Comment (douglas-j-bothwell) [reply docs/security-testing-orchestration/sto-techref-category/shared/step_palette/target/auto-detect/_note.md]: @atmollo-harness **Ingestion** or **Extraction**. My bad. I've updated this.

**Review by atmollo-harness — DISMISSED** [review]

File: docs/security-testing-orchestration/use-sto/orchestrate-and-ingest/extraction-scans.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+---
+title: Extraction mode for SaaS scan tools
+description: Extract and ingest results from SaaS scanners
+sidebar_label: Extraction scans
+sidebar_position: 35
+---
+
+Extraction scans can be useful when you're working with SaaS-based scanners such as [SonarQube](/docs/security-testing-orchestration/sto-techref-category/sonarqube-sonar-scanner-reference) and [Burp Enterprise](/docs/security-testing-orchestration/sto-techref-category/burp-scanner-reference), where you can extract scan results from an API endpoint. 
+
+There most common use cases for Extraction mode are:
```
Comment (GabeAboy) [inline docs/security-testing-orchestration/use-sto/orchestrate-and-ingest/extraction-scans.md:10]:
```suggestion
The most common use cases for Extraction mode are:
```

File: docs/security-testing-orchestration/use-sto/orchestrate-and-ingest/extraction-scans.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+   Suppose you have a scan job defined in your SaaS instance that automatically scans a target whenever it gets updated. In this case, you can use Extraction mode to ingest the latest scan results for that target.
+
+2) Extract results from a custom scan.
+
+   - Orchestration mode is useful if you're running a default scan in the SaaS instance.
+   - Extraction mode is useful when you have a custom scan job defined in the SaaS instance. An example pipeline might look something like this:
+
+     1) A Run step sends a request to the SaaS API that starts the custom scan.
+     2) A scan step, such as SonarQube or Veracode, runs in Extraction mode. When the custom scan finishes, the step extracts the results from the scanner API and correlates, deduplicates, and ingests the results.
```
Comment (GabeAboy) [inline docs/security-testing-orchestration/use-sto/orchestrate-and-ingest/extraction-scans.md:22]: At the top you mentioned Sonarqube and Burp extraction, here you mention SonarQube and Veracode. Is this intentional?

**Comment by douglas-j-bothwell** [comment]
Closing due to numerous merge conflicts as a result of the doc reorg. Continuing this work in this PR: https://github.com/harness/developer-hub/pull/7171

**Pull request closed without merging**